validation/td-validation.ttl

```diff
@@ -367,15 +367,31 @@ well-known types necessary to implement the WoT interaction model described in
 operations types SHOULD NOT be arbitrarily set by servients
 </span>.
 </p>
+
+<p>
+The optional <code>response</code> name-value pair can be used to provide metadata, such as content type information, that is only itended for the reponse messages. 
+<span class="rfc2119-assertion" id="td-expectedResponse-default-contentType">
+If no <code>response</code> name-value pair is provided it MUST be assumed that the content type of the 
+response is equal to the assigned content type of the form.</span>
+E.g., if the value of the content type of the form is <code>application/xml</code> the assumed value of the content type of the response will be also <code>application/xml</code>.
```

Comment:
Before the "For instance", best add the hint:
Note that contentType within an ExpectedResponse Class does not have a Default Value.